devops/providers/azure-devops/templates/infrastructure/README.md

```diff
@@ -0,0 +1,204 @@
+
+Here's some more [guidance](https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#protecting-branches) on leveraging Azure DevOPS build validation checks with protected branch's.
+
+### Published Artifact
+
+The Terraform workspace directory is published as a pipeline build [artifact](https://docs.microsoft.com/en-us/azure/devops/pipelines/artifacts/build-artifacts?view=azure-devops&tabs=yaml#how-do-i-publish-artifacts) following the automated test step(s). This artifact ensures that the same manifest is applied across all release stages.
+
+## Release Stages
+
+### QA
+
+This environment is a lighter weight replica of staging and intended for manual quality assurance checkouts. Automated terraform tests are asserted against this environment, similar to DevInt.
+
+This Azure Devops stage does not support parallel releases due to the dedicated resources.
+
+#### Release Gate
+
+There's a gate prior to applying the execution plan onto the QA environment. The technical operator should review the terraform execution plan prior to approving the gate request.
+
+#### Stage Completion
+
+The Staging release will be automatically triggered upon a successful completion / release of this stage.
+
+### Staging
+
+The staging environment is a 1:1 replica of the production environment.
+
+Cobalt automated tests should be tagged to application environment targets to avoid automated tests that happen to mutate data in production.
+
+This Azure Devops stage does not support parallel releases due to the dedicated resources.
+
+#### Release Gate
+
+There's a gate prior to applying the execution plan onto the Staging environment. The technical operator should review the terraform execution plan prior to approving the gate request.
+
+### Production
+
+The production stage release is auto triggered upon feature branch merges into master. This stage will apply the desired terraform changes within your production environment.
+
+One git scm flow worth considering is the [GitHub flow](https://guides.github.com/introduction/flow/). This model promotes short feedback loops for feature branch pull requests. This means work branches (‘work’ could mean a new feature or a bug fix – there is no distinction) starts from the production code (master) and are short lived.
+
+Our pipeline can support any scm git workflow(gitflow, etc).
+
+The master branch is a record of known good production code. Feature branch(s) should only be merged into master once: 1) code review's are approved 2) all CD pre-prod releases are complete. These can be validated as part of your master branch policy.
+
+This Azure Devops stage does not support parallel releases due to the dedicated resources.
+
+## Pipeline setup
+
+As discussed earlier, there's one git repository for each infrastructure template, as technical operators are code owners of cloud resource manifests.
+
+### Add Baseline Azure Build Pipeline
```

Comment:
This is helpful but IMO we could probably link to existing documentation here as its not something we are proposing to do differently

Reply:
The only reason I added the screenshots was because the azDO felt a bit unclear on the pipeline creation steps.